src/Arcus.Templates.Tests.Integration/appsettings.json

```diff
@@ -10,7 +10,8 @@
         "AuthKey": "#{Arcus.TestInfra.EventGrid.Auth.Key}#"
       },
       "ServiceBus": {
-        "ConnectionStringWithQueue": "#{Arcus.ServiceBus.ConnectionStringWithQueue}#"
+        "ConnectionStringWithQueue": "#{Arcus.ServiceBus.ConnectionStringWithQueue}#",
+        "ConnectionStringWithTopic": "#{Arcus.ServiceBus.ConnectionStringWithTopic}#" 
```

Comment:
Did you update the build/variable group?

Reply:
This key was already there 😄 probably for the Arcus.Messaging testing.

Reply:
Perfect, thanks!